File: luogu/python/P1089.py
n = 0
m = 0
hand = 0
mother = 0
m_index = 0
judge = 0
while (n<12):
    m = int(input())
    if m<=(300+hand):
        temp = (300-m) + hand
        while (temp>=100):
            temp -= 100
            mother += 100
        hand = temp
    else:
        judge+=1
        if judge==1:
            m_index = (n+1)
    n+=1
if judge != 0:
    m_index = -m_index
    print(m_index)
else:
    print(int(mother*(1.2))+hand)

File: luogu/P1089.java
import java.io.*;
import java.util.*;

public class P1089{
    public static void main(String[] args){
        Scanner sc = new Scanner(System.in);
        int judge = 0;
        int m_index = 0;
        int hand = 0;
        int mother = 0;
        for (int i=0;i<12;i++){
            int m = sc.nextInt();
            if (m<=300+hand){
                int temp = 300-m+hand;
                while(temp>=100){
                    temp-=100;
                    mother+=100;
                }
                hand = temp;
            }
            else{
                judge++;
                if(judge==1){
                    m_index = (i+1);
                }
            }
        }
        if(judge!=0){
            m_index = -m_index;
            System.out.print(m_index);
        }
        else{
            System.out.print((int)(mother*1.2+hand));
        }

    }
}